dynamodb/src/it/scala/zio/dynamodb/LiveSpec.scala

```diff
@@ -200,6 +200,27 @@ object LiveSpec extends DefaultRunnableSpec {
           withDefaultTable { tableName =>
             getItem(tableName, PrimaryKey(id -> "nowhere", number -> 1000)).execute.map(item => assert(item)(isNone))
           }
+        },
+        testM("batch get item") {
```

Comment:
This was actually supposed to just be a batch get instead of a batch get transaction. I noticed we didn't have an integration test for the batch gets.